autonomy/fsm/scaffold/scaffold_skill.py

```diff
@@ -180,14 +180,19 @@ def get_actual_abstract_round_abci_package_public_id(
         package_manager = PackageManagerV1.from_dir(Path(ctx.registry_path))
         packages = [
             package_id.public_id
-            for package_id in package_manager.dev_packages.keys()
-            if package_id.author == ABSTRACT_ROUND_SKILL_PUBLIC_ID.author
-            and package_id.name == ABSTRACT_ROUND_SKILL_PUBLIC_ID.name
-            and package_id.package_type == PackageType.SKILL
+            for package_id in [
```

Comment:
@angrybayblade 
you can combine multiple iters in one with: itertools.chain(iter1, iter2)